solutions/mean.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+A module for finding the mean of a list of numbers.
+
+Module contents:
+    - mean: finds the mean of a list of numbers.
+
+Created on XX XX XX
```

Comment:
To improve clarity, it would be helpful if you could update the date when this was created.